cdptools/research_utils/transcripts.py

```diff
@@ -78,7 +160,7 @@ def download_most_recent_transcripts(
     save_dir.mkdir(parents=True, exist_ok=True)
 
     # Get most recent transcript data
-    most_recent = get_most_recent_transcript_manifest(db)
+    most_recent = get_transcript_manifest(db=db, order_by_field=order_by_field)
 
     # Begin storage
     most_recent.apply(
```

Comment:
I know this is not a apart of the changes, but is it possible to do a threaded download file here too?

Reply:
Huh. I really thought `dataframe.apply` was already threaded for some reason. Will _gladly_ make the change. :+1: